src/evergreen/performance_results.py

```diff
@@ -30,23 +28,31 @@ def __init__(self, test_result, api):
         """Create an instance of a test run."""
         super(PerformanceTestRun, self).__init__(test_result, api)
 
+    @property
+    def start(self):
+        """Get the start time for the given test run."""
```

Comment:
[suggestion]: do we want to mention (in the followon comment) that the value format varies?

Reply:
good idea, updated.